File: go/lesson03/exercise/formatter/formatter.go
package main

import (
	"fmt"
	"log"
	"net/http"
)

func main() {
	http.HandleFunc("/format", func(w http.ResponseWriter, r *http.Request) {
		helloTo := r.FormValue("helloTo")
		helloStr := fmt.Sprintf("Hello, %s!", helloTo)
		w.Write([]byte(helloStr))
	})

	log.Fatal(http.ListenAndServe(":8081", nil))
}

File: python/lesson03/exercise/formatter.py
from flask import Flask
from flask import request

app = Flask(__name__)
 
@app.route("/format")
def format():
    hello_to = request.args.get('helloTo')
    return 'Hello, %s!' % hello_to
 
if __name__ == "__main__":
    app.run(port=8081)
